liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -158,13 +158,17 @@ private void generateFeatures() throws MojoExecutionException, PluginExecutionEx
         generatedFiles.add(GENERATED_FEATURES_FILE_NAME);
 
         Set<String> existingFeatures = getServerFeatures(servUtil, generatedFiles, optimize);
+        Set<String> nonCustomFeatures = new HashSet<String>(); // binary scanner only handles actual Liberty features
+        for (String feature : existingFeatures) { // custom features are "usr:feature-1.0" or "myExt:feature-2.0"
+            if (!feature.contains(":")) nonCustomFeatures.add(feature);
```

Comment:
How confident are we that we should remove all features with a ":"?

Reply:
Pretty confident. I checked with YK and he agreed. Also other code in LMP does this. [E.g. https://github.com/OpenLiberty/ci.common/blob/70db23360b84405b0ad48cab752d495d9cd3a99b/src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java#L345](https://github.com/OpenLiberty/ci.common/blob/70db23360b84405b0ad48cab752d495d9cd3a99b/src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java#L345-L372)